src/App.css

```diff
@@ -1 +1,16 @@
+.active {
+  background: #fff;
+  border: 1px solid green;
+  border-bottom: none;
+}
 
+.link {
+  text-decoration: none;
+  color: #000;
+  margin-bottom: -1px;
+}
+
+.border-bottom {
```

Comment:
css classes shouldn't be named as styles inside

Reply:
even if I decided to use atomic methodology of naming?